webservices/common/models.py

```diff
@@ -43,6 +43,7 @@ class Candidate(BaseModel):
     party_full = db.Column(db.String(255))
     state = db.Column(db.String(2))
     name = db.Column(db.String(100))
+    committees = db.relationship('CandidateCommitteeLink', backref='candidates')
```

Comment:
I actually deleted these lines by accident in an earlier PR--this commit is reverting to the earlier version. They're going to go away with #648, but we need them for now.